account_mass_reconcile/tests/test_reconcile.py

```diff
@@ -0,0 +1,68 @@
+# © 2014-2016 Camptocamp SA (Damien Crier)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+from odoo import fields, exceptions, tools
+from odoo.modules import get_module_resource
+
+
+class TestReconcile(common.TransactionCase):
+
+    def setUp(self):
```

Comment:
we can use SavepointCase + setUpClass to avoid creating records again and again on each test.